grai-server/app/api/mutations.py

```diff
@@ -46,7 +48,9 @@ async def createConnection(
         namespace: str,
         name: str,
         metadata: JSON,
-        secrets: JSON,
+        secrets: Optional[JSON],
```

Comment:
`secrets`, `schedules`, and `is_active` are marked optional but don't have a default value. Does it make sense to provide a default `None` value to the argument or did you have in mind that the caller ought to specifically provide `None` when those values weren't available?

Reply:
Fascinating question, I was assuming that the GraphQL engine would be providing a `None` value, but I haven't exactly checked.  Would be good for me to write a test over one of these to see what happens.